batch_job/src/amazon_ec2.sh

```diff
@@ -0,0 +1,156 @@
+#!/bin/sh
```

Comment:
The file that created is not this file. The file that is created is `_temp_amazon_ec2_script.sh`. This file is where that file comes from. I kept this file in case any changes needed to be made to the generated script. Editing the generated file in C (specifically dealing with escaping and newlines) is a big pain, so I wrote a perl script `convert.pl` that will pull from the `amazon_ec2.sh` file, escape it, and print it into the variable in the `batch_job_amazon.c` file.